gobblin-core/src/main/java/gobblin/publisher/TimePartitionedDataPublisher.java

```diff
@@ -43,30 +36,4 @@
   public TimePartitionedDataPublisher(State state) throws IOException {
     super(state);
   }
-
-  /**
-   * This method needs to be overridden for TimePartitionedDataPublisher, since the output folder structure
-   * contains timestamp, we have to move the files recursively.
-   *
-   * For example, move {writerOutput}/2015/04/08/15/output.avro to {publisherOutput}/2015/04/08/15/output.avro
-   */
-  @Override
-  protected void addWriterOutputToExistingDir(Path writerOutput, Path publisherOutput, WorkUnitState workUnitState,
```

Comment:
As far as I could tell the main difference with the TimePartitionedDataPublisher and the BaseDataPublisher is:
  1. TimePartitionedDataPublisher _recursively_ listed all files under the writerOutputDir and moved them to the publisherOutputDir
  2. BaseDataPublisher listed all files under the writerOutputDir and moved them to the publisherOutputDir

BaseDataPublisher now copies the files as specified in `ConfigurationKeys.WRITER_FINAL_OUTPUT_FILE_PATHS`.  Because of this, there is no need to list the files (recursively or otherwise) within the writerOutputDir.